response_operations_ui/controllers/reporting_units_controllers.py

```diff
@@ -56,8 +56,8 @@ def generate_new_enrolment_code(collection_exercise_id, ru_ref):
 
 def resend_verification_email(party_id):
     logger.debug('Re-sending verification email', party_id=party_id)
-    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/reporting-unit/resend-verification-email/{party_id}'
-    response = requests.post(url)
+    url = f'{app.config["PARTY_URL"]}/party-api/v1/resend-verification-email/{party_id}'
+    response = requests.get(url, auth=app.config['PARTY_AUTH'])
```

Comment:
Out of scope for this change but a get action feels wrong for this api in party. Possibly need to raise a TBL to change this?

Reply:
@tej99 tech debt item created for this to be considered 👍